scripts/governance-add-proposal.js

```diff
@@ -16,12 +16,12 @@ async function transact() {
       tx.uintCV(43100),
       tx.uintCV(900),
       tx.stringUtf8CV('Add STX/WELSH Pair'),
```

Comment:
I notice that this still says 'Add STX/WELSH Pair'.

Reply:
Fixe